python/read_json.py

```diff
@@ -1,9 +1,15 @@
+"""Read JSON file from stdin line by line"""
 import json
 
-file_path = "person-records.json"
+json_input = ""
 
-with open(file_path, "r") as file:
-    data = json.load(file)
+while True:
```

Comment:
@nicholaides I prob should've put this in a different PR because it's for a different test, but is this how we want to handle testing reading the JSON file?